src/wallet/wallet.cpp

```diff
@@ -2529,6 +2537,7 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
                 BOOST_FOREACH (const CRecipient& recipient, vecSend)
                 {
                     CTxOut txout(recipient.asset, recipient.nAmount, recipient.scriptPubKey);
+                    txout.nNonce.vchCommitment = std::vector<unsigned char>(recipient.confidentiality_key.begin(), recipient.confidentiality_key.end());
```

Comment:
`createrawtransaction` sets the confidentiality keys of the outputs to be the CT address pubkey. Typically they stay there until `BlindTransaction`, but in the `fundrawtransaction` path you simply lose those keys since these newly made outputs are written back in FundTransaction.